crates/iota-indexer/migrations/pg/2025-03-13-081552_optimistic_transactions/down.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS optimistic_tx_kinds;
```

Comment:
The indexes should also be dropped.

Reply:
From the docs it seems they are dropped automatically:

`DROP TABLE always removes any indexes, rules, triggers, and constraints that exist for the target table.`

https://www.postgresql.org/docs/current/sql-droptable.html

So I would rather prefer to keep the migration shorter.